pkg/global/synapseconstants.go

```diff
@@ -22,6 +22,8 @@ const (
 	Local                 = true
 	MaxConnectionAttempts = 10
 	ExecutionLogsPath     = "/var/log/synapse"
+	PingWait              = 30 * time.Second
```

Comment:
what will happen after ping wait timeout ? will it retry from syanpse side .

Reply:
It will wait for ping for 30 s , if no ping arrived during that duration the connection will get timed out and retry  will happen